src/lib/reservationTimes.ts

```diff
@@ -83,15 +83,20 @@ const maxEnd = (stns: SettingsManager, dateStr: string, cat: ReservationCategory
 const nRes = (stns: SettingsManager, dateStr: string, cat: ReservationCategory) =>
 	Math.floor((maxEnd(stns, dateStr, cat) - minStart(stns, dateStr, cat)) / inc(stns, dateStr));
 
-export const startTimes = (stns: SettingsManager, dateStr: string, cat: ReservationCategory) =>
-	Array(nRes(stns, dateStr, cat))
+export const getStartEndTimes = (
```

Comment:
returns an array of all possible start/end times, e.g. [9:00, 9:30, ..., 19:30, 20:00]